### PR TITLE
Make deep_qa pip installable

### PR DESCRIPTION
**Pull request by DeNeutoy**

### CODE REVIEW

**Comment by DeNeutoy** [comment]
This is ready for review - I've simplified the readme given that we now have the docs (and the wall of text of the previous one I'm not actually sure was super helpful). I've also checked that this works by installing it in a virtualenv and everything seems to work correctly. To do this, run `python setup.py sdist' and pip install the resulting `deep_qa-0.1.tar.gz` file which will appear under the `/dist` directory.

**Comment by matt-gardner** [comment]
Thanks again for getting this set up.  I will look at this soon - tomorrow might be busy working on finalizing a couple of drafts to be submitted to EMNLP, but I will look at this if I get time.  I still want to figure out how to get this to work with markdown instead of RST in the README; I'll look into it when I look over this PR.

File: README.md [hunk 1]
```diff
@@ -1,127 +1,161 @@
...
-- The attentive reader, from [Teaching Machines to Read and Comprehend](https://www.semanticscholar.org/paper/Teaching-Machines-to-Read-and-Comprehend-Hermann-Kocisk%C3%BD/2cb8497f9214735ffd1bd57db645794459b8ff41), by Hermann and others
-- Windowed-memory MemNNs, from [The Goldilocks Principle: Reading Children's Books with Explicit Memory Representations](https://www.semanticscholar.org/paper/The-Goldilocks-Principle-Reading-Children-s-Books-Hill-Bordes/1ee46c3b71ebe336d0b278de9093cfca7af7390b) (in progress)
-
-As well as some of our own, as-yet-unpublished variants.  There is a lot of similarity between the models in these papers, and our code is structured in a way to allow for easily switching between these models.
-For a description of how we've built an extensible memory network architecture in this library, see [this readme.](./src/main/python/deep_qa/models/memory_networks/README.md)
-# Datasets
+Deep QA
+=======
```
Comment (matt-gardner) [inline README.md]: Sections and sub-sections need to be switched back to markdown still.

**Comment by matt-gardner** [comment]
This looks great.  I'm just going to update the README a bit myself, too, before merging this.

**Pull request merged**